burner-wallet-plugin/tokenbridge-bw-exchange/src/utils/abis/HomeBridgeNativeToErc.ts

```diff
@@ -0,0 +1,1036 @@
+export default [
```

Comment:
I see the ABI is taken as is and it is huge. Could we re-use only methods required by the plugin?

Reply:
Updated the abis for `HomeBridgeNativeToErc` and `ForeignBridgeNativeToErc` to only include the methods/events needed